test/test_translator.py

```diff
@@ -138,6 +117,36 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_invalid_data() :
```

Comment:
Each of these asserts seems to be testing a valid unit. We should break those out into appropriatly named unit tests of their own.

Reply:
added valid test function name and pushed into dev.